lib/src/controllers/user_controller.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:shared_preferences/shared_preferences.dart';
+import '../services/constants.dart' as global;
+
+class UserController {
+  late SharedPreferences _prefs;
+
+  Future<bool> get isUserLoggedIn async {
+    SharedPreferences prefs = await SharedPreferences.getInstance();
```

Comment:
This is temporary, I guess? :)

Reply:
yes. future we can use direct firebase login validation.